app/resources/css-default.template

```diff
@@ -628,7 +628,7 @@ img.icon {
 
 .view input.add-note {
   width: 100%;
-  margin-top: 10px; /* Vertically align blue buttom with record card */
+  margin-top: 10px;  /* To vertically align the "Add note" button with the record card */
```

Comment:
I checked the other css code and i found that there is one also over 80 chars
![css](https://cloud.githubusercontent.com/assets/15961861/18821510/de9be566-83e1-11e6-96bf-cb6bd9d87e66.png)

Should I fix it too?